inspirehep/config.py

```diff
@@ -307,14 +307,16 @@
 # SEARCH_ELASTIC_KEYWORD_MAPPING -- this variable holds a dictionary to map
 # invenio keywords to elasticsearch fields
 SEARCH_ELASTIC_KEYWORD_MAPPING = {
-    "author": ["authors.full_name"],
-    "exactauthor": ["exactauthor.raw"],
+    "author": ["authors.full_name", "authors.alternative_name"],
+    "exactauthor": ["exactauthor.raw", "authors.full_name",
+                    "authors.alternative_name"
+                    ],
     "abstract": ["abstracts.value"],
     "collaboration": ["collaboration", "collaboration.raw^2"],
     "collection": ["_collections"],
     "doi": ["dois.value"],
     "doc_type": ["facet_inspire_doc_type"],
-    "affiliation": ["authors.affiliations.value"],
+    "affiliation": ["authors.affiliations.value", "corporate_author"],
```

Comment:
Because of this https://docs.google.com/spreadsheets/d/1Mm5FWjQV4V5pp3pQkRZ3LHxNJTAGSD_122huIaJFN7s/edit#gid=0 . 
I am adding the fields that are in the sheet and it was there.